### PR TITLE
appointment - vuex

### PR DESCRIPTION
**Pull request by viktorChubyrka**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -1,10 +1,49 @@
 import Vue from "vue";
```
Comment (user4i) [inline src/store.js]: Please convert store to vuex modules

**Review by user4i — CHANGES_REQUESTED** [review]

**Comment by user4i** [comment]
Please rename store `appoints` to  `appointments`

File: src/store/appoint.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import Axios from "axios";
```
Comment (user4i) [inline src/store/appoint.js]: Please rename store `appoints` to  `appointments`

File: src/store/appoint.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+const appointmentsUrl = "http://localhost:3000/appointments";
+const clientsUrl = "http://localhost:3000/clients";
+export default {
+  state: {
+    appoints: [],
+    clients: []
```
Comment (user4i) [inline src/store/appoint.js]: `clients` cannot be in `appointments` store

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/store/appoint.js (hunk 3 above)
Comment (viktorChubyrka) [reply src/store/appoint.js]: done

File: src/store/appoint.js (hunk 2 above)
Comment (viktorChubyrka) [reply src/store/appoint.js]: done

File: src/store.js (hunk 1 above)
Comment (viktorChubyrka) [reply src/store.js]: done

File: src/config.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export default {
+  clientsUrl: "http://localhost:3000/clients",
```
Comment (user4i) [inline src/config.js]: In config should be only 1 url
```
apiUrl = "http://localhost:3000"
```
and inside stores you can code at the begging of file
```
const baseUrl = `${config.apiUrl}/clients`
const baseUrl = `${config.apiUrl}/appointments`
```

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/views/Appointments.vue [hunk 5]
```diff
@@ -44,70 +44,79 @@
...
+      date: new Date().toISOString().substr(0, 10),
+      time: ""
     };
   },
-  created() {
-    this.fetchClients();
-    this.fetchAppoints();
+  computed: {
+    clientsList() {
+      return this.$store.getters.allClients;
```
Comment (user4i) [inline src/views/Appointments.vue]: Please rename 
`this.$store.getters.allClients;`
to
`this.$store.getters.CLIENTS;`

too as `APPOINTS`

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/config.js (hunk 4 above)
Comment (viktorChubyrka) [reply src/config.js]: done

File: src/views/Appointments.vue (hunk 5 above)
Comment (viktorChubyrka) [reply src/views/Appointments.vue]: done

**Pull request merged**